build/subrake.rb

```diff
@@ -4,6 +4,17 @@
 
 task :all => :default
 
+task :rakefile do
+    verbose(VERBOSE)
+    COMPONENTS.each do |e|
+        Dir.chdir(e) do
+          if File.exists? "project.json"
+              sh "corto genrakefile"
```

Comment:
Please follow the pattern that other tasks use, by forwarding to the project rakefile (`sh rake -f #{e}/rakefile`)